tsconfig.json

```
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    /**
+     * Must allow:
+     */
+    "downlevelIteration": true,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "noUnusedParameters": false,
+    "removeComments": false,
+    "incremental": true,
+
+    /**
+     * Nice to have:
+     */
+    // "preserveConstEnums": true,
+    // "declaration": true,
+    // "allowUnreachableCode": false,
+    // "pretty": true,
+
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "target": "es5",
+    "lib": ["es2015", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "@aws-sdk/*": ["packages/*/src"],
+      "@aws-sdk/clients-*": ["clients/clients-*/"]
```

The package names are prefixed with `client-` and not `clients-`
Example: [@aws-sdk/client-dynamodb](https://www.npmjs.com/package/@aws-sdk/client-dynamodb)

```suggestion
      "@aws-sdk/client-*": ["clients/clients-*/"]
```

---

Ahh yes, good catch